charts/orchestrator/values.schema.json

```diff
@@ -1012,6 +1012,16 @@
                                     "cpu": "500m"
                                 }
                             }]
+                        },
```

Comment:
Should we include this in the schema, if these properties appear only in the values-rc.yaml and not in the values.yaml?

Reply:
anyway, if that's a must to fix the blocker, there is a need to bump the version of the chart.

Reply:
It's not needed to fix the issue, but I found it missing first and I thought it was the root cause.
We can close it and I will amend the operator as well so that this is now showing in the schema but it's available in the spec as optional and empty

Reply:
ok, so let's close this one, since all of the context of values-rc.yaml isn't represented in the values.schema.json